app/models/company_value.rb

```diff
@@ -1,3 +1,5 @@
 class CompanyValue < ApplicationRecord
+  has_many :kudos, foreign_key: 'company_value', inverse_of: :company_value, dependent: :destroy
```

Comment:
You do not have to specify those values, if the names of the associations are matching the name of the classes 
```suggestion
  has_many :kudos, dependent: :destroy
```